### DISCUSSION TITLE
Zoom MP4 Feature

### DISCUSSION THREAD

**Discussion by alanfermtz** [sircharlo/meeting-media-manager#1789] [Q&A] [answered]
What is the difference between sharing media mp4 and original media?

**Comment by mtdvlpr** [comment, marked as answer]
When sharing through M3, there is no difference, the mp4 file that's been created from an image will be presented as an image. The purpose of the `Convert media to MP4 format` setting is to share those mp4 files through Zoom's share video feature. Normally, you'd only use that setting for fully remote meetings. See [our documentation](https://sircharlo.github.io/meeting-media-manager/en/#configuration) for more details.

**Reply by mtdvlpr** [reply]
Did that answer your question @alanfermtz?
